index.ios.js

```diff
@@ -0,0 +1,13 @@
+//This module is only for logging the events using Fabric Answers
+'use strict';
+
+var FabricAnswers = require('react-native').NativeModules.FabricAnswers;
+var FAnswers = {
+    logView(attributes) {
+            FabricAnswers.logViewEvent(attributes)
```

Comment:
@ravirdv  I guess , we can remove this file , it should be common across the platform.